docs/user-guide/deployments/deploy-on-kubernetes/greptimedb-operator-management.md

```diff
@@ -0,0 +1,218 @@
+# GreptimeDB Operator Management
+
+## Prerequisites
+
+- Kubernetes >= v1.18.0
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) >= v1.18.0
+- [Helm](https://helm.sh/docs/intro/install/) >= v3.0.0
+
+## Quick Start
+
+:::warning
+It's not recommended to use this method to install the GreptimeDB Operator in a production environment. 
+:::
+
+The fastest way to install the GreptimeDB Operator is to use `bundle.yaml`:
+
+```bash
+kubectl apply -f \
+  https://github.com/GreptimeTeam/greptimedb-operator/releases/latest/download/bundle.yaml \
+  --server-side
+```
+
+The greptimedb-operator will be installed in the `greptimedb-admin` namespace. When the `greptimedb-operator` is running, you can use the following command to verify the installation:
+
+```bash
+kubectl get pods -n greptimedb-admin
+```
+
+<details>
+  <summary>Expected Output</summary>
+```bash
+NAME                                   READY   STATUS    RESTARTS   AGE
+greptimedb-operator-7947d785b5-b668p   1/1     Running   0          2m18s
+```
+</details>
+
+## Production Deployment
+
+For production deployments, it's recommended to use Helm to install the GreptimeDB Operator.
+
+### Installation
+
+You can refer [Install the GreptimeDB Operator](/user-guide/deployments/deploy-on-kubernetes/getting-started.md#install-the-greptimedb-operator) for detailed instructions.
+
+### Upgrade
+
+We always publish the latest version of the GreptimeDB Operator as a Helm chart in our [official Helm repository](https://github.com/GreptimeTeam/helm-charts/tree/main).
+
+When the new version of the GreptimeDB Operator is released, you can upgrade the GreptimeDB Operator by running the following commands.
+
+#### Update the Helm repository
+
+```bash
+helm repo update greptime
+```
+
+<details>
+<summary>Expected Output</summary>
+```bash
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "greptime" chart repository
+Update Complete. ⎈Happy Helming!⎈
+```
+</details>
+
+You can use the following command to search the latest version of the GreptimeDB Operator:
+
+```bash
+helm search repo greptime/greptimedb-operator
+```
+
+:::note
+The output may vary depending on the version of the charts.
+:::
+
+
+<details>
+<summary>Expected Output</summary>
+```bash
+NAME                        	CHART VERSION	APP VERSION  	DESCRIPTION
+greptime/greptimedb-operator	0.2.9        	0.1.3-alpha.1	The greptimedb-operator Helm chart for Kubernetes.
+```
+</details>
+
+:::tip
+If you want to list all the available versions, you can use the following command:
+
+```
+helm search repo greptime/greptimedb-operator --versions
+```
+:::
+
```

Comment:
please do not use tip for the listing versions, move it before `Upgrade the GreptimeDB Operator` to check all the available versions.